assets/style.css

```diff
@@ -58,3 +58,8 @@ nav {
 .kpi__icon {
   max-height: 3rem; 
 }
+
+.test{

```

Comment:
Can you change it to a more descriptive class name?

Reply:
change it to circle